.github/workflows/package-client.yml

```diff
@@ -126,6 +126,9 @@ jobs:
         run: npm run web:release
         env:
           PARSEC_APP_SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          # Configure the base href of the WebApp,
+          # we use a relative path since we want to provide a bundle that does not enforce a way to distribute the app.
+          PARSEC_APP_BASEHREF: ./
```

Comment:
Do we have a reason to make this configurable ?

My concern here is we are going to to end up with different default depending on if we generate the release from the CI or with a `npm run web:release` run from the dev machine (which will lead to some headache on why the release fails when hosted from the server...)

Reply:
Do you prefer that we change the default used in `vite.config.ts` ?